src/index.js

```diff
@@ -120,28 +131,28 @@ const createCurrentDataSet = () => {
       searchableData = bookingData;
       roomInfo = roomData;
 
-      // create current user
-      // const today = '2020/02/02';
       const date = new Date().toISOString();
       const dateStr = date.split('T');
       inputDateSelector.setAttribute('min', dateStr[0])
       let today = dateStr[0].split('-').join('/')
-      const currentUser = new User(userData)
 
-      document.getElementById('navDate').innerHTML = `${today}`
-      userNameSelector.setAttribute('data-userId', currentUser.id)
-      userNameSelector.innerHTML = `Hi, ${currentUser.name}`
+      if (userData !== undefined) {
```

Comment:
How does userData work? where is the get/login functionality for that, and what are you doing with that in promise.all above?

Reply:
It's really only fetching the user name, if they're a customer. Otherwise (in the else block), the manager doesn't need access to the same actions that I use for customer. but I think I'll need to use that in my search for a customer.